rest_framework_swagger/views.py

```diff
@@ -97,17 +97,10 @@ def handle_permission_denied(self, request):
 
 
 class SwaggerResourcesView(APIDocView):
-    renderer_classes = (JSONRenderer,)
+    renderer_classes = (JSONRenderer, )
 
     def get(self, request):
-        apis = []
-
-        resources = [resource for resource in self.get_resources()
-                     if self.handle_resource_access(request, resource)]
-
-        for path in resources:
-            apis.append({'path': '/%s' % path, })
-
+        apis = [{'path': '/' + path} for path in self.get_resources()]
```

Comment:
Because David Beazley told me it's ok.

![python_cookbook__3rd_edition_pdf__page_77_of_706_](https://cloud.githubusercontent.com/assets/203574/8628351/3044c128-2720-11e5-9288-41b1def6a283.png)

But in any event, the string formatting in this file is pretty schizophrenic and `'%s' % value` is less favourable to `'{}'.format(value)` or `{0}.format(value)` [says python docs](https://docs.python.org/2/library/string.html#format-string-syntax).